tests/Xamarin.Forms-Performance-Integration/Environment.txt

```diff
@@ -0,0 +1 @@
+debug.mono.log gref
```

Comment:
This should use an `=`:

```
debug.mono.log=gref
```

For example: https://github.com/xamarin/xamarin-android/blob/27ec4988b74160e6a20efb24f3377f05af072b07/tests/EmbeddedDSOs/EmbeddedDSO/Environment.txt#L1